projects/scripts/adi_board.tcl

```diff
@@ -719,17 +719,17 @@ proc ad_cpu_interrupt {p_ps_index p_mb_index p_name} {
     set p_net [get_bd_nets -of_objects [get_bd_pins sys_concat_intc_0/In$p_index]]
     set p_pin [find_bd_objs -relation connected_to [get_bd_pins sys_concat_intc_0/In$p_index]]
```

Comment:
The $p_pin variable should be just the actual pin of the interrupt concatenation module:
set p_pin [get_bd_pins sys_concat_intc_0/In$p_index]

This should solve the connection issues.